pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -85,6 +85,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   cmakeFlags = [
     "-DLLAMA_NATIVE=OFF"
     "-DLLAMA_BUILD_SERVER=ON"
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
This gets added automatically by nix, you can see the flag twice in the build logs with this added flag.

Reply:
Thank you!